builtin/provisioners/chef/linux_provisioner_test.go

```diff
@@ -120,7 +120,26 @@ func TestResourceProvider_linuxInstallChefClient(t *testing.T) {
 
 			Commands: map[string]bool{
 				"curl -LO https://www.chef.io/chef/install.sh": true,
```

Comment:
We should update this URL to be `https://omnitruck.chef.io/install.sh` (along with all other spots `www.chef.io` is being used in test and no-test files).